homeassistant/components/tractive/sensor.py

```diff
@@ -132,6 +137,17 @@ async def async_added_to_hass(self):
         device_class=DEVICE_CLASS_BATTERY,
         entity_class=TractiveHardwareSensor,
     ),
+    TractiveSensorEntityDescription(
+        key=ATTR_LIVE_TRACKING_REMAINING,
+        name="Live tracking remaining time",
+        native_unit_of_measurement=TIME_SECONDS,
+        entity_class=TractiveHardwareSensor,
+    ),
+    TractiveSensorEntityDescription(
+        key=ATTR_TRACKER_STATE,
```

Comment:
Do we know what states this sensor can have? If so, we can add a custom device class and make these states translatable.

Reply:
So far I have only seen 'NOT_REPORTING' and 'OPERATIONAL'.
Do you have any examples of how to do that?